app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-3" style="text-align: center; padding: 2rem">
```

Comment:
Please separate HTML from CSS. Every styling should be defined in `.scss` files (or plain `.css`, whatever you prefer).

This should be applied to every similar occurrence in this PR.